plugins/sonarqube/README.md

```diff
@@ -17,15 +17,15 @@ yarn add @backstage/plugin-sonarqube
 2. Add plugin to the app:
 
 ```js
-// packages/app/src/plugins.ts
+# packages/app/src/plugins.ts
```

Comment:
Think this breaks the formatting in the markdown as they are set as `javascript` comments. We prefer to keep `//` javascript comment.

Reply:
@benjdlambert , Okay.
I've addressed your comment.